Language files en_US -> de_DE (FINISHED), minor language file edits

keraldi opened the pull request:
en_Us.lang 
* had some lines that read "Undocumented", added lines for those 
* added a line for Water Ritual (that might or might not be implemented similar to Lava ritual for Raw Will).
* added documentation for the Ritual of Expulsion
* fixed some typos
* added a typo at the Water Ritual, fixed it again
* clarified Mobs -> Entities for the Ritual of Regeneration
* fixed some wrong names or explainations due to typos/wrong names and changed mechanics

de_DE.lang
* translated the whole en_US.lang including the changes I made to it.

Commit:
Fixed downgrade level inconsistency

keraldi committed:
Squashed commit of the following:

commit 7263e6fbc2a3411167a10fa7a931fb9a74832286
Author: PiscesdanAT <40119554+PiscesdanAT@users.noreply.github.com>
Date:   Sat Jun 9 18:01:30 2018 +0200

    Update de_DE.lang (#1337)

    Someone mixed up dusk and dawn

Updated de_DE.lang
Added some missing stuff in en_US.lang

keraldi committed:
bloodmagic/.../de_DE.lang

 * more fitting translations

bloodmagicguide/.../de_DE.lang

 * 1/3 finished

.. en_US.lang

Whitespaces

Signed-off-by: Iorce <tgremeyer@icloud.com>

keraldi committed:
Update en_US.lang

Fixed the typo I inserted with previous commit.

Iorce commented:
Did some work on de_DE and en_US. Comment for push.

keraldi committed:
Merge branch '1.12' into 1.12

keraldi commented:
For "The Demon Kin" is it as in "The family of demons" (Demonkind => Dämonenartige) or "The children of demons" (Kinder der Dämonen (I would translate it as "Lilith's children" => "Liliths Kinder"))?

For "Well of Suffering" is it as in "The source of suffering" or "Fountain (A well in a village etc.) of suffering"?

For "Feathered Knife" is it as in "A knife with feathers attached to it"/"with a texture/structure that is similar to that of a feather" or "resilient, springy knife"?

WayofTime commented:
"Demon Kin" means the child of demons.

Well of Suffering is closer to the first one, though it is harder to translate the full meaning of sayings/phrases.

Feathered Knife is mainly describing the fact that the knife does not kill you, just damages you. It's not a knife with feathers attached to it, so I suppose the last description you have is the most accurate.

keraldi commented:
Are the teachers formal or informal with their students?
(In German, formal speech uses 2nd person plural, while informal speech uses 2nd person singular, they're the same in English (see: I, you, he/she/it, we, you, they)).

EDIT: Also, I personally don't like the sound of "Haupt-Ritualstein" (direct translation would be main ritual stone), I carried it over to my translations. Using "Meisterritualstein" would keep the abbreviation the same ("MRS" for both English and German) and it rolls off the tongue a bit better, too.

This should or would carry over to the Routing Nodes as well.

So should I change it to "Meisterritualstein" or keep "Haupt-Ritualstein"?

EDIT2: One occurence of Soul Network in the old language files translated it with "LP-Netzwerk", however I don't like putting that into the Guidebook translation. Should I use "Blood Network" ("Blutnetzwerk"), or "Soul Network" ("Seelennetzwerk")?

EDIT3: While I translated the bloodmagic/.../de_DE.lang manually, I've since switched to a machine learning translator (http://deepl.com/) for the guidebook translation (because it would take too many man hours otherwise) and I'm only reviewing that, errorchecking and making minor adjustments. There might be some differences in style/variation, which I'll try to keep constant and if I'm unsure, I'll reply with the possible translations and exact meanings.

keraldi commented:
Is the tinkerer in the lava ritual story a man or a woman?
This<sup>is<sup>very<sub>important!</sub></sup></sup>

EDIT: I'm obviously not paying attention. It's a male.

keraldi committed:
Merge remote-tracking branch 'origin/1.12' into 1.12

# Conflicts:
#	src/main/resources/assets/bloodmagic/lang/de_DE.lang
#	src/main/resources/assets/bloodmagic/lang/en_US.lang

keraldi commented:
Entry: guide.bloodmagic.entry.ritual.wellOfSuffering.info in bloodmagicguide/.../en_US.lang:

"Damages all living non-player entities" Does it damage Zombies, Skeletons etc.? 
They're undead. I suppose the entry is incorrect. Well, they are living as in existing but are technically not living due to being undead. Not sure how I feel about it, might review it and change it a couple times.

keraldi commented:
Cannot find an entry for Soul Fray in the bloodmagic/.../en_US.lang (therefore it isn't in any). This means the Soul Fray debuff won't have a translation (even in bloodmagicguide/.../xx_XX.lang)

keraldi commented:
How should breaking of the fourth wall in the guidebook be handled (i.e.: "... the ritual makes no assumptions about the thing it is harvesting - it can only harvest mod-added crops if the mod in question registers a harvesting handler with the mod.") ?

I would feel personally attacked trying to translate "... mod in question registers a harvesting handler with the mod." - It feels like it doesn't belong there and nobody who is able to code would use translated terminology.

TehNut commented:
Yeah I don't really think that wording should be there

keraldi commented:
I would rewrite the beginning of the entry for the Ritual of Expulsion to something like this:

"As Blood Magic is a delicate art, you might find yourself in a situation in which you want to prevent the intrusion of other individuals/players/the common folk. Though worldly means exist in the form of simple traps and highly technological advancements like force fields, these can still be circumvented by some of the more crafty kind. The Ritual of Expulsion offers a more final solution that should give you the peace you need to conduct your experiments through the use of teleportation magic! ... "

In order to get rid of the fourth wall. If it is undesirable (at the very least it should not mention server....).

keraldi committed:
bloodmagic/.../de_DE.lang:
- improved translations - again

bloodmagicguide/.../en_US:
- mudpack -> modpack (someone doesn't like them, it seems)
- wares off -> wears off (Typo)

bloodmagicguide/.../de_DE:
- another batch of translations

Signed-off-by: Iorce <tgremeyer@icloud.com>

keraldi committed:
bloodmagic/.../de_DE.lang:
- improved translations - again

bloodmagicguide/.../en_US:
- mudpack -> modpack (someone doesn't like them, it seems)
- wares off -> wears off (Typo)

bloodmagicguide/.../de_DE:
- another batch of translations
- finished the "Ritual Master"

Signed-off-by: Iorce <tgremeyer@icloud.com>

keraldi committed:
bloodmagic/.../de_DE.lang:
- improved translations - a bit more literal for some while keeping variety (Suppression and Displacement can have the same translation)

bloodmagicguide/.../de_DE:
- translated architect entry texts / titles

keraldi committed:
bloodmagicguide/.../en_US.lang:
consistency (was Sacrificial Knife, every occurrence of that item is now called "Sacrifical Dagger"

bloodmagicguide/.../de_DE.lang:
some more....

keraldi committed:
bloodmagicguide/.../en_US.lang:
corrected required Blood Altar tier from "greater than" to "greater than or equal to"

bloodmagicguide/.../de_DE.lang:
some more....

keraldi committed:
bloodmagicguide/.../en_US.lang:
added clarification for Incense Altar block range

bloodmagicguide/.../de_DE.lang:
changed "Wahrheitssiegel" to "Siegel der Weissagung"
changed "Weihrauchaltar" to "Räucheraltar"
some more....

keraldi committed:
bloodmagicguide/.../en_US.lang:
clarification for Sigil of the Fast Miner entry ("and" -> "at")

bloodmagicguide/.../de_DE.lang:
some more....

keraldi committed:
bloodmagicguide/.../de_DE.lang:
some more....

keraldi committed:
Changed "Blutorb" to "Blutkugel" in translation. Might have missed some cases of severe german grammar.

bloodmagicguide/.../en_US.lang:
Removed a "my", if someone misses it, it myght be readded.

bloodmagicguide/.../de_DE.lang:
some more....

keraldi committed:
long overdue: closes #1320 (the disorient thing that nobody cares about)
bloodmagicguide/.../en_US.lang:
  people are now less warry and more wary about sentient mimics.

bloodmagicguide/.../de_DE.lang:
  some more.... (just the mimic stuff, since I commit after every typo I fix :o )

keraldi committed:
bloodmagicguide/.../de_DE.lang:
some more....

Completed "The Architect"

keraldi committed:
bloodmagic/.../de_DE.lang:
Leere Tafel -> Leere Schiefertafel (material clarification, remaining names unchanged to keep the names short)
Rudimentäre Falle -> [...] Schlinge (item icon looks like the latter, word has multiple meanings)
Ansammlung Kristalle [...] -> Kirstallansammlung (easier on the tongue, better text flow)

bloodmagicguide/.../en_US.lang:
fixed mistake in textflow

bloodmagicguide/.../de_DE.lang:
some more....

keraldi committed:
bloodmagic/.../de_DE.lang:
  clarification
  Haupt- -> Meister

bloodmagicguide/.../de_DE.lang:
  some more....

bloodmagicguide/.../en_US.lang:
  Updated entry to reflect the new mechanic for creating pure will types

keraldi committed:
bloodmagic/.../de_DE.lang:
  Ansammlung -> Gruppe

bloodmagicguide/.../de_DE.lang:
finished!

bloodmagicguide/.../en_US.lang:
fixed ritual name

keraldi commented:
That's all for now (Unless someone decides the guide needs more content).

Arcaratus commented on src/main/resources/assets/bloodmagicguide/lang/de_DE.lang:
```diff
@@ -0,0 +1,270 @@
+guide.bloodmagic.page.lp=LP: %d
+guide.bloodmagic.page.minimumWill=Minimaler Wille: %f
+guide.bloodmagic.page.drainedWill=Verbrauchter Wille: %f
+guide.bloodmagic.shapelessOrb=Formloses Orb-Rezept
```
Blutkugel here instead of Orb

Arcaratus commented on src/main/resources/assets/bloodmagic/lang/de_DE.lang:
```diff
@@ -70,74 +70,150 @@ item.bloodmagic.baseComponent.reagentWater.name=Wasserreagenz
+item.bloodmagic.cutting_fluid.basic.name=Einfache Schneidelösung
+item.bloodmagic.cutting_fluid.explosive.name=Explosives Pulver
+
+item.bloodmagic.demonCrystal.default.name=Kristall dämonischen Willens
```
"Willens" is a bad translation. Think something more like willpower and volition

keraldi replied:
Do you just want to change it for the crystals, everything base (i.e. Crystal of demonic Willpower, Raw/Corrosive/Destructive/Vengeful/Steadfast Crystal of demonic Willpower, Demonic Willpower, Raw/Corrosive/Destructive/Vengeful/Steadfast demonic Willpower)?

Or every single entry of "demonic Will" in any form?

keraldi replied:
"Willens" is a special grammar case of "Wille" both refer to the will or the "I want this" of a Person. "Willenskraft" would be the proper translation for "Willpower", however nobody would use it in such an extensive context. It would be too long and too repetitive.

In any case, I want further confirmation before changing ALL entries (because it's a lot).

keraldi replied:
from google translate: 

  | der Wille | will, intention

And the quality of that "intention" is the power it has.
Or so I think.

Arcaratus replied:
Although it does have that meaning, it's a bit obscure. I think it's just easier to translate it to "Will" as a borrowed noun.

keraldi replied:
Leaving it untranslated would make it even more obscure because people won't expect an untranslated word there, especially if it has meaning in German.

Translating it as "Willenskraft" /"Will power" for basic things (The will drops/crystals) would make the meaning clear while preventing the exhaustive spam of long words and repetitions if every single entry would be translated.

Arcaratus replied:
Sounds good.

Arcaratus commented on src/main/resources/assets/bloodmagic/lang/de_DE.lang:
```diff
@@ -70,74 +70,150 @@ item.bloodmagic.baseComponent.reagentWater.name=Wasserreagenz
+item.bloodmagic.cutting_fluid.explosive.name=Explosives Pulver
+
+item.bloodmagic.demonCrystal.default.name=Kristall dämonischen Willens
+item.bloodmagic.demonCrystal.corrosive.name=Kristall ätzenden dämonischen Willens
```
I find "korrosiv" better than "ätzenden"

keraldi replied:
Changed it!

Arcaratus commented on src/main/resources/assets/bloodmagic/lang/de_DE.lang:
```diff
@@ -70,74 +70,150 @@ item.bloodmagic.baseComponent.reagentWater.name=Wasserreagenz
+item.bloodmagic.sentientPickaxe.name=Empfindsame Spitzhacke
+item.bloodmagic.sentientShovel.name=Empfindsame Schaufel
+
+item.bloodmagic.nodeRouter.name=Knoten-Router
```
"Knoten" is a bad translation for node. Use "Netzknoten" instead.

keraldi replied:
In CS, we translate "node" in graphs as "Knoten" or "Ecke" (the latter meaning corner which I sure as hell won't use), "Netzknoten" would be a possible (and proper) translation, but again, nobody would actually refer to it with the "long" name and just refer it with a shortened version in context instead.

It is also implied that it is a network because you use those things in a network that transfers stuff. I'll change it for the tool (Node-Router) but I want further confirmation before changing it for the other stuff.

Arcaratus commented on src/main/resources/assets/bloodmagic/lang/de_DE.lang:
```diff
@@ -70,74 +70,150 @@ item.bloodmagic.baseComponent.reagentWater.name=Wasserreagenz
+item.bloodmagic.itemFilter.exact.name=Präziser Gegenstandsfilter
+item.bloodmagic.itemFilter.ignoreNBT.name=Ignoriere NBT Gegenstandsfilter
+item.bloodmagic.itemFilter.modItems.name=Mod Gegenstandsfilter
+item.bloodmagic.itemFilter.oreDict.name=Ore Dictionary Gegenstandsfilter
```
See if there's a German translation of Ore Dictionary somewhere

keraldi replied:
Since the Ore Dictionary is a Forge thing and you'd rarely see someone insane enough to translate stuff that is used in programming itself, I doubt that I'll find one. An initial search didn't yield any results.

And the translation is "Erzwörterbuch" which sounds stupid.

Arcaratus commented on src/main/resources/assets/bloodmagic/lang/de_DE.lang:
```diff
@@ -222,56 +455,295 @@ tooltip.bloodmagic.diviner.duskRune=Runen der Abenddämmerung: %d
+tooltip.bloodmagic.ghost.everything=Alles
+tooltip.bloodmagic.ghost.amount=Geistergegenstandsanzahl: %d
+
+tooltip.bloodmagic.currentType.default=Contains: Roher Wille
```
Translate the "Contains:"

keraldi replied:
Done!

Arcaratus commented on src/main/resources/assets/bloodmagic/lang/de_DE.lang:
```diff
@@ -286,25 +758,141 @@ chat.bloodmagic.ritual.prevent=Das Ritual versucht, dir zu widerstehen.
+jei.bloodmagic.recipe.drainRate=Ablauf: %d LP/t
+jei.bloodmagic.recipe.minimumSouls=Minimum: %1$..2f Wille
+jei.bloodmagic.recipe.soulsDrained=Verbraucht: %1$..2f Wille
+jei.bloodmagic.recipe.lpDrained=Drained: %.d LP
```
Same here

keraldi replied:
Done!

Commit:
Finalized de_DE.lang

Ätzend -> Korrosiv

Contains: -> Enthält:

Commit:
Finalized de_DE.lang

ätzend -> korrosiv

Commit:
Update de_DE.lang

Commit:
Update de_DE.lang

Drained -> Verbraucht
Item -> Gegenstand

Commit:
Update de_DE.lang

Knoten-Router -> Netzknoten-Router

Commit:
Update de_DE.lang

Changed the base names for demonic Will to "Willpower"

-> clarification for the user that the residue is actualy willpower and the quality of said will is a measure of the potential

-> not changing other occurrences or notions in the book as no sane person would repeat "willpower" at every occasion.

keraldi committed:
Applied quick fixes to de_DE.lang

keraldi commented:
Updated to incorporate Arcaratus' quick fixes. Please review and merge :3

keraldi committed:
Merge remote-tracking branch 'origin/1.12' into 1.12

TehNut commented on gradle.properties:
```diff
@@ -4,9 +4,7 @@ mod_version=2.2.12
 mc_version=1.12.2
 forge_version=14.23.2.2611
 curse_id=224791
-
```
Remove the changes to this file.

TehNut commented on src/main/java/WayofTime/bloodmagic/livingArmour/downgrade/LivingArmourUpgradeDisoriented.java:
```diff
@@ -9,7 +9,7 @@
 public class LivingArmourUpgradeDisoriented extends LivingArmourUpgrade {
     public static final int[] costs = new int[]{-10, -20, -30, -40, -70, -80, -100, -140, -180, -220};
-    public static final double[] chance = new double[]{0.001, 0.002, 0.003, 0.004, 0.005, 0.006, 0.008, 0.0010, 0.0012, 0.014};
+    public static final double[] chance = new double[]{0.001, 0.002, 0.003, 0.004, 0.005, 0.006, 0.008, 0.010, 0.012, 0.014};
```
This change should be moved to it's own PR

Commit:
Merge me!
reverted a whitespace change in gradle.properties (no idea where that came from) and moved the disorient thingy to another PR)

Commit:
Take No.2

at fixing whitespaces.

Commit:
TAKE NO.3

(HOPEFULLY FINAL TAKE)
